python/graphstorm/gconstruct/transform.py

```diff
@@ -507,7 +604,17 @@ def parse_label_ops(confs, is_node):
     label_conf = confs[0]
     assert 'task_type' in label_conf, "'task_type' must be defined in the label field."
     task_type = label_conf['task_type']
-    # By default, we use all labels for training.
+    if 'custom_split' in label_conf:
+        custom_split = label_conf['custom_split']
+        assert len(custom_split) == 3, \
+                "Custom data split needs to provide train/val/test index."
+        train_idx = np.load(custom_split[0])
```

Comment:
Can any of train/val or test be none?
For example, in inference, only test_idx is needed.